src/Cake.SqlServer/ExtractDacpacSettings.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using Cake.Core.IO;
+
+namespace Cake.SqlServer
+{
+    /// <summary>
+    /// Settings for extract dacpac from database
+    /// </summary>
+    public class ExtractDacpacSettings
+    {
+        private readonly List<Tuple<string, string>> _tables = new List<Tuple<string, string>>();
+        private string _outputFile;
```

Comment:
Please remove underscores from private var names - to be consistent with the rest of the project